portal/server/nunjucks-configuration/filter-localiseTimestamp.ts

```diff
@@ -0,0 +1,20 @@
+import { formatInTimeZone } from 'date-fns-tz';
+
+export const filterLocaliseTimestamp = (
+  utcTimestamp: number | string,
```

Comment:
Since this function is in TS now, might be ideal to define a `type` `utcTimestamp`?

Reply:
IMO this would make the code less readable - I think `number | string` is simple enough to be used inline